contrib/hack/test_walkthrough.sh

```diff
@@ -135,6 +135,10 @@ retry -n 10 \
 
 echo 'Waiting on pods to come up...'
 
+wait_for_expected_output -x -e 'Pending' -n 10 \
```

Comment:
@pmorie It's waiting for none of the the pods to have a `Pending` state. The `-x` flag for this function makes it wait for the absence of the provided string. Similarly, 142 is waiting for the absence of `ContainerCreating`.

After none of the pods are in either of these states, we then check to see if each of them are actually `Ready` on line 146.